HW_1/task2.py

```diff
@@ -0,0 +1,12 @@
+#Задача 2: Найдите сумму цифр трехзначного числа. 
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0)
+
+print ('Введите трехзначное число')
+number=int(input())
+sum = 0
+while number > 0:
+    digit = number % 10
+    sum += digit
```

Comment:
Если переменная используется единожды, то ее можно в принципе не использовать.
```suggestion
    sum += number % 10
```